src/features/user/use_cases.rs

```diff
@@ -38,11 +47,16 @@ pub async fn vote(app_ctx: &AppContext, vote: Vote) -> Result<(), UserError> {
 /// Gets votes for a snap with the given ID from a given [`ClientHash`]
 ///
 /// [`ClientHash`]: crate::features::user::entities::ClientHash
+#[allow(unused_must_use)]
 pub async fn get_snap_votes(
     app_ctx: &AppContext,
     snap_id: String,
     client_hash: String,
 ) -> Result<Vec<Vote>, UserError> {
+    // Ignore but log warning, it's not fatal
+    update_category(app_ctx, &snap_id)
+        .await
+        .map_err(|e| warn!("{}", e));
```

Comment:
[With 1.76.0](https://blog.rust-lang.org/2024/02/08/Rust-1.76.0.html) there's now a `inspect` on Result + Option which might be more intuitive than over .map_err

Reply:
Good catch, it does mean we have to be sure GHA will be using `1.76`, but this is much better. I think there are other places in the code that do this but that's a separate PR.

Reply:
Yep for another PR. We can set the rust-version in cargo.toml then.